java-practice-methods/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/java-practice-methods.iml" filepath="$PROJECT_DIR$/.idea/java-practice-methods.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
Unnecessary file (this is the project settings file for IntelliJ IDEA).
Fix it:
- create file .gitignore
- add to the file .idea/